README.md

```diff
@@ -6,10 +6,22 @@ QA Tools for Odoo maintainers (MQT)
 
 The goal of Maintainer Quality Tools (MQT) is to provide helpers to ensure the quality of Odoo addons.
 
-If you are a developer and are looking to run the pre-commit action locally,
+If you are a developer and are looking to run the pre-commits action locally,
 you don't need this repository.
-Follow the [pre-commit tool](https://pre-commit.com/#install) installations instructions.
-The OCA-recommended `pre-commit` configuration and associated linters configuration files may be found in the [sample_files](https://github.com/OCA/maintainer-quality-tools/tree/master/sample_files/pre-commit-13.0) directory of this repo.
+Follow the [pre-commit tool](https://pre-commit.com/#install) installation instructions.
+
+In a nutshell:
+
+- Install pre-commit in your system: ``pip3 install pre-commit``
```

Comment:
```suggestion
- Install pre-commit in your system: ``pip3 install --user pre-commit``
```

Recommending `--user` to avoid user that get a permission issue to retry with sudo and break their system.